app/src/components/domain/RiskImminentEvents/Pdc/index.tsx

```diff
@@ -128,22 +203,73 @@ function Pdc(props: Props) {
             const stormPositions = (storm_position_geojson as unknown as unknown[] | undefined)
                 ?.filter(isValidPointFeature);
 
+            const cycloneFiveDays = (cyclone_five_days_cou as unknown as unknown[] | undefined)
+                ?.filter(isValidFeature);
+
+            const cycloneThreeDays = (cyclone_three_days_cou as unknown as unknown[] | undefined)
+                ?.filter(isValidFeature);
             // forecast_date_time : "2023 SEP 04, 00:00Z"
             // severity : "WARNING"
             // storm_name : "HAIKUI"
             // track_heading : "WNW"
             // wind_speed_mph : 75
 
-            const geoJson: GeoJSON.FeatureCollection<GeoJSON.Geometry> = {
+            const geoJson: GeoJSON.FeatureCollection<GeoJSON.Geometry, EventGeoJsonProperties> = {
                 type: 'FeatureCollection' as const,
                 features: [
                     footprint ? {
                         ...footprint,
                         properties: {
-                            ...footprint.properties,
+                            alertType: getAlertType(footprint?.properties?.severity),
                             type: 'exposure',
                         },
                     } : undefined,
+
+                    ...cycloneFiveDays?.map(
+                        (feature) => ({
+                            ...feature,
+                            properties: {
+                                alertType: getAlertType(feature?.properties?.severity),
```

Comment:
Is there any reason we are adding `alertType` property here?

Reply:
to calculate which type of alert level is this and what color to use on visualization

Reply:
What does the `cycloneFiveDays` represent here? Is it the cone of uncertainty for five days?

Reply:
@puranban is `alertType` associated with uncertainty as well? Isn't it only associated with the buffers, nodes ?